zoo_api/server/boot/root.js

```diff
@@ -1,7 +1,39 @@
+const bodyParser = require('body-parser');
+
+const jsonParser = bodyParser.json();
+
+const loginFunc = (Account, Role) => (req, res) => {
```

Comment:
it'd be nice to have messages regarding missing login info before a login request is made.
```suggestion
const loginFunc = (Account, Role) => (req, res) => {
  if(!req.body.email){
    res.status(401).json({ message: 'Email address missing. Please enter email address.', status: 401 });
  } else if(!req.body.password) {
    res.status(401).json({ message: 'Password missing. Please enter password.', status: 401 });
  } else {
  // do the rest of the account login
```